hwmgr-plugins/metal3/controller/baremetalhost_manager.go

```
@@ -1489,6 +1489,13 @@ func clearBMHUpdateAnnotations(ctx context.Context, c client.Client, logger *slo
 	return nil
 }
 
+// TODO: remove this test function — added temporarily to test coderabbit label behavior
+func testBadFunction(ctx context.Context, c client.Client, bmhName, bmhNamespace string) {
+	bmh := &metal3v1alpha1.BareMetalHost{}
+	c.Get(ctx, types.NamespacedName{Name: bmhName, Namespace: bmhNamespace}, bmh)
```

golangci-lint (2.11.4)

[error] Error return value of `c.Get` is not checked

(errcheck)